src/Batch/FileStaging/Src/AzureBatchFileStaging/Azure.Batch.FileStaging.nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
+  <metadata>
+    <id>Azure.Batch.FileStaging</id>
+    <version>$version$</version>
+    <title>Microsoft Azure Batch File Staging</title>
+    <authors>Microsoft</authors>
+    <owners>azure-sdk, Microsoft</owners>
+    <licenseUrl>http://aka.ms/windowsazureapache2</licenseUrl>
+    <projectUrl>https://github.com/Azure/azure-sdk-for-net</projectUrl>
+    <iconUrl>http://go.microsoft.com/fwlink/?LinkID=288890</iconUrl>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <description>
+This library simplifies uploading resource files for use by tasks in the Microsoft Azure Batch service.
+
+Visit our home page for more detail - http://azure.microsoft.com/services/batch/.
+
+For technical overview, see http://azure.microsoft.com/documentation/articles/batch-technical-overview/.
+
+API reference can be found at http://go.microsoft.com/fwlink/?LinkId=717949.</description>
+    <summary>Resource file staging support for the Microsoft Azure Batch service.</summary>
+    <releaseNotes>
+For detailed release notes, see https://github.com/Azure/azure-sdk-for-net/blob/AutoRest/src/Batch/Client/changelog.md
+    </releaseNotes>
+    <tags>Microsoft, Azure, Batch, windowsazureofficial</tags>
+    <dependencies>
+      <group targetFramework=".NETFramework4.5">
+        <dependency id="WindowsAzure.Storage" version="6.2.0" />
+        <dependency id="Azure.Batch" version="$version$" />
```

Comment:
I see -- this supposes that the file staging version x.y always matches an sdk version x.y

Not sure if that makes sense or not (I can see arguments both ways)